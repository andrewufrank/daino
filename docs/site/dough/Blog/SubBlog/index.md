---
title: index for subdir 
author: AUF
date: Jan. 4, 2019
keywords: test
abstract: The subdirectory experiment
pageTemplate: page3.yaml
indexPage: True
indexSort: title 
---

an index page for test 



