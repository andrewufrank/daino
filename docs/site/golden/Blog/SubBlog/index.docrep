Docrep {meta1 = MetaPage {dyFn = "/home/frank/Workspace11/ssg/docs/site/dough/Blog/SubBlog/index.md", dyLink = "Blog/SubBlog/index.md", dyLang = DLenglish, dyTitle = "index for subdir", dyAbstract = "The subdirectory experiment", dyAuthor = "AUF", dyDate = Just "Jan. 4, 2019", dyKeywords = "test", dyBibliography = Nothing, dyStyle = Nothing, dyReferences = Nothing, dyNoCite = Nothing, dyPublish = Nothing, dyIndexPage = True, dyIndexSort = Just "title", dyIndexEntry = IndexEntry {fn = "/home/frank/Workspace11/ssg/docs/site/dough/Blog/SubBlog/index.md", link = "Blog/SubBlog/index.md", title = "index for subdir", abstract = "The subdirectory experiment", author = "AUF", date = "Jan. 4, 2019", publish = Nothing, indexPage = True, dirEntries = [], fileEntries = []}}, pan1 = Pandoc (Meta {unMeta = fromList [("abstract",MetaInlines [Str "The",Space,Str "subdirectory",Space,Str "experiment"]),("author",MetaInlines [Str "AUF"]),("date",MetaInlines [Str "Jan.",Space,Str "4,",Space,Str "2019"]),("indexPage",MetaBool True),("indexSort",MetaInlines [Str "title"]),("keywords",MetaInlines [Str "test"]),("pageTemplate",MetaInlines [Str "page3.yaml"]),("title",MetaInlines [Str "index",Space,Str "for",Space,Str "subdir"])]}) [Para [Str "an",Space,Str "index",Space,Str "page",Space,Str "for",Space,Str "test"]]}