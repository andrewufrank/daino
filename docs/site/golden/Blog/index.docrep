Docrep {meta1 = MetaPage {dyFn = "/home/frank/Workspace11/ssg/docs/site/dough/Blog/index.md", dyLink = "Blog/index.md", dyLang = DLenglish, dyTitle = "primary index for Blog", dyAbstract = "The directory for experiments.", dyAuthor = "AUF", dyDate = Just "Jan. 4, 2019", dyKeywords = "test", dyBibliography = Nothing, dyStyle = Nothing, dyReferences = Nothing, dyNoCite = Nothing, dyPublish = Nothing, dyIndexPage = True, dyIndexSort = Just "reverseDate", dyIndexEntry = IndexEntry {fn = "/home/frank/Workspace11/ssg/docs/site/dough/Blog/index.md", link = "Blog/index.md", title = "primary index for Blog", abstract = "The directory for experiments.", author = "AUF", date = "Jan. 4, 2019", publish = Nothing, indexPage = True, dirEntries = [], fileEntries = []}}, pan1 = Pandoc (Meta {unMeta = fromList [("abstract",MetaInlines [Str "The",Space,Str "directory",Space,Str "for",Space,Str "experiments."]),("author",MetaInlines [Str "AUF"]),("date",MetaInlines [Str "Jan.",Space,Str "4,",Space,Str "2019"]),("indexPage",MetaBool True),("indexSort",MetaInlines [Str "reverseDate"]),("keywords",MetaInlines [Str "test"]),("pageTemplate",MetaInlines [Str "page3.yaml"]),("title",MetaInlines [Str "primary",Space,Str "index",Space,Str "for",Space,Str "Blog"])]}) [Para [Str "an",Space,Str "index",Space,Str "page",Space,Str "for",Space,Str "Blog"]]}