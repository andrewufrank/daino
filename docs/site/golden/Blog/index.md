---
title: primary index for Blog 
author: AUF
date: Jan. 4, 2019
keywords: test
abstract: The directory for experiments.
pageTemplate: page3.yaml
indexPage: True 
indexSort: reverseDate
---

an index page for Blog 




