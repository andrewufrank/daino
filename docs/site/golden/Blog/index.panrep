Panrep {panyam = MetaPage {dyFn = "/home/frank/Workspace11/ssg/docs/site/dough/Blog/index.md", dyLink = "Blog/index.md", dyLang = DLenglish, dyTitle = "primary index for Blog", dyAbstract = "The directory for experiments.", dyAuthor = "AUF", dyDate = Just "Jan. 4, 2019", dyKeywords = "test", dyBibliography = Nothing, dyStyle = Nothing, dyReferences = Nothing, dyNoCite = Nothing, dyPublish = Nothing, dyIndexPage = True, dyIndexSort = Just "reverseDate", dyIndexEntry = IndexEntry {fn = "/home/frank/Workspace11/ssg/docs/site/dough/Blog/index.md", link = "Blog/index.md", title = "primary index for Blog", abstract = "The directory for experiments.", author = "AUF", date = "Jan. 4, 2019", publish = Nothing, indexPage = True, dirEntries = [IndexEntry {fn = "/home/frank/Workspace11/ssg/docs/site/dough/Blog/SubBlog/index.md", link = "Blog/SubBlog/index.md", title = "index for subdir", abstract = "The subdirectory experiment", author = "AUF", date = "Jan. 4, 2019", publish = Nothing, indexPage = True, dirEntries = [], fileEntries = []}], fileEntries = [IndexEntry {fn = "/home/frank/Workspace11/ssg/docs/site/dough/Blog/blog1.md", link = "Blog/blog1.md", title = "Mein erster Blog", abstract = "Ein Blog ohne Sinn auf Deutsch", author = "FILL AUTHOR", date = "2020-06-18", publish = Nothing, indexPage = False, dirEntries = [], fileEntries = []},IndexEntry {fn = "/home/frank/Workspace11/ssg/docs/site/dough/Blog/withRef.md", link = "Blog/withRef.md", title = "postWithReference", abstract = "What are the objects of ontologists .", author = "FILL AUTHOR", date = "2010-07-29", publish = Nothing, indexPage = False, dirEntries = [], fileEntries = []},IndexEntry {fn = "/home/frank/Workspace11/ssg/docs/site/dough/Blog/postwk.md", link = "Blog/postwk.md", title = "postwk with image", abstract = "A silly text not needing an abstract.", author = "AUF", date = "Jan. 4, 2019", publish = Nothing, indexPage = False, dirEntries = [], fileEntries = []}]}}, panpan = Pandoc (Meta {unMeta = fromList [("abstract",MetaInlines [Str "The",Space,Str "directory",Space,Str "for",Space,Str "experiments."]),("author",MetaInlines [Str "AUF"]),("date",MetaInlines [Str "Jan.",Space,Str "4,",Space,Str "2019"]),("indexPage",MetaBool True),("indexSort",MetaInlines [Str "reverseDate"]),("keywords",MetaInlines [Str "test"]),("pageTemplate",MetaInlines [Str "page3.yaml"]),("title",MetaInlines [Str "primary",Space,Str "index",Space,Str "for",Space,Str "Blog"])]}) [Para [Str "an",Space,Str "index",Space,Str "page",Space,Str "for",Space,Str "Blog"]]}