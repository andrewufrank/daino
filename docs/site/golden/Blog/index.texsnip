TexSnip {snipyam = MetaPage {dyFn = "/home/frank/Workspace11/ssg/docs/site/dough/Blog/index.md", dyLink = "Blog/index.md", dyLang = DLenglish, dyTitle = "primary index for Blog", dyAbstract = "The directory for experiments.", dyAuthor = "AUF", dyDate = Just "Jan. 4, 2019", dyKeywords = "test", dyBibliography = Nothing, dyStyle = Nothing, dyReferences = Nothing, dyNoCite = Nothing, dyPublish = Nothing, dyIndexPage = True, dyIndexSort = Just "reverseDate", dyIndexEntry = IndexEntry {fn = "/home/frank/Workspace11/ssg/docs/site/dough/Blog/index.md", link = "Blog/index.md", title = "primary index for Blog", abstract = "The directory for experiments.", author = "AUF", date = "Jan. 4, 2019", publish = Nothing, indexPage = True, dirEntries = [IndexEntry {fn = "/home/frank/Workspace11/ssg/docs/site/dough/Blog/SubBlog/index.md", link = "Blog/SubBlog/index.md", title = "index for subdir", abstract = "The subdirectory experiment", author = "AUF", date = "Jan. 4, 2019", publish = Nothing, indexPage = True, dirEntries = [], fileEntries = []}], fileEntries = [IndexEntry {fn = "/home/frank/Workspace11/ssg/docs/site/dough/Blog/blog1.md", link = "Blog/blog1.md", title = "Mein erster Blog", abstract = "Ein Blog ohne Sinn auf Deutsch", author = "FILL AUTHOR", date = "2020-06-18", publish = Nothing, indexPage = False, dirEntries = [], fileEntries = []},IndexEntry {fn = "/home/frank/Workspace11/ssg/docs/site/dough/Blog/withRef.md", link = "Blog/withRef.md", title = "postWithReference", abstract = "What are the objects of ontologists .", author = "FILL AUTHOR", date = "2010-07-29", publish = Nothing, indexPage = False, dirEntries = [], fileEntries = []},IndexEntry {fn = "/home/frank/Workspace11/ssg/docs/site/dough/Blog/postwk.md", link = "Blog/postwk.md", title = "postwk with image", abstract = "A silly text not needing an abstract.", author = "AUF", date = "Jan. 4, 2019", publish = Nothing, indexPage = False, dirEntries = [], fileEntries = []}]}}, unTexSnip = "an index page for Blog"}