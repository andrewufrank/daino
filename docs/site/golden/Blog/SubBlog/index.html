<!DOCTYPE html>
<!-- the master page for the pandoc templating mechanism
	needs page-title, page-title-postfix
		author date and keywords for indexing
    contentHtml, beforeContent, afterContent to wrap htmlcontent  
	uses style.css as default, other can be loaded with key css 
	-->
<html lang="DLenglish">
  <head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" />


  <style type="text/css">
      code{white-space: pre-wrap;}
      span.smallcaps{font-variant: small-caps;}
      span.underline{text-decoration: underline;}
      div.column{display: inline-block; vertical-align: top; width: 50%;}
	  </style>

	
	
          <meta name="author" content="AUF" />
              <meta name="dcterms.date" content="Jan. 4, 2019" />
              <meta name="keywords" content="test" />
    
	<title></title>
	
        <style>
            body {
            margin: 0;
            font-family: Arial, Helvetica, sans-serif;
            }

            .hero-image {
            background-image: url('');
            background-color: white;
            height: 200px;
            background-position: center;
            background-repeat: no-repeat;
            background-size: cover;
            position: relative;
            }

            .hero-text {
            transform: translate(-50%, -50%);
            color: white;
            text-shadow: 2px 2px 8px black;
            font-weight: bold;
            vertical-align: top;
            }
        </style>
    </head>

    <body>
    <!-- AF - my template for pandoc page (from the default) page3.dtpl -->
        

    <section class="hero">
        <div class="hero-image">
            
                <h1></h1>
                <h3></h3>
        <div/>
    </section>
    <section class="header">

            <section class="menu">
		<ul>
            			</ul>
    </section>
    <section>
                <header>
        <h1 class="title">index for subdir</h1>
        
                <p class="author">AUF</p>
                </header>
         
    </section>

    <section class=menu2>
		<ul>
            			</ul>
    </section>

        
<!-- new content section -->
    
    The subdirectory experiment 
    
    


        <p class=tiny>
        Produced with SGG  on  from  with master4.dtpl.
        </p>

            </body>


</html>
